README.md

```diff
@@ -1 +1,184 @@
-# HTML-CSS-JavaScript-capstone-project---Conference-page
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>First Microverse Capstone Project</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [📖 Video Description](#vid-desc)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Conference Page <a name="about-project"></a>
+
+**Conference page** is the first Microverse Capstone at the end of Module 1
+
+## Video Description <a name="vid-desc">
+[Loom video describing project](#)
```

Comment:
- [ ]  Please, kindly ensure you've recorded a loom video with the following features





- The maximum length of the video is 5 minutes.
- The video contains a demo of the project, showing the pages in different screen sizes.
- The video contains a highlight about a piece of the code the you find interesting or is proud of making.

Reply:
Link of the [https://www.loom.com/share/69349eb9e8de4197988d23b173c3f82b]

Reply:
- [ ] Good job so far but your video length is too long. Not that the maximum length of the video required for this project is 5 mins. Please fix it in order to follow the requirement :+1: